### ISSUE TITLE
Icons, bold, italics?

### ISSUE THREAD

**Issue by jkamenik** [dundalek/atom-markdown-mindmap#50]
Seems like it isn't supported it would be nice to render icons as well as bold and italic text.  I usually use representative icons to quickly visualize similar to important info.  Bold to call attention to a phrase, and italic to denote a "comment".

Like this

```markdown
# Header

- ![alt text][shield] some important note
- some **bolded** note
  - *interesting fact*

[shield]: https://github.com/adam-p/markdown-here/raw/master/src/common/images/icon48.png "shield"
```

**Comment by dundalek** [comment]
Yeah, only plain text is supported inside nodes. This request is similar to https://github.com/dundalek/atom-markdown-mindmap/issues/35. The tricky part is mixing HTML with SVG, but it might be possible using [foreignObject](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject).
